frontend/src/app/home/footer.component.html

```diff
@@ -0,0 +1,42 @@
+<footer id="footer" role="contentinfo" class="usa-footer usa-footer-slim">
+  <div class="usa-footer-primary-section">
+    <div class="usa-grid">
+      <ul class="usa-unstyled-list fs-footer-links">
+        <li class="fs-footer-item"><a href="http://www.usda.gov/wps/portal/usda/usdahome">USDA.gov</a></li>
+        <li class="fs-footer-item"><a href="http://www.usda.gov/wps/portal/usda/usdahome?navtype=FT&amp;navid=PRIVACY_POLICY">Privacy Policy</a></li>
+        <li class="fs-footer-item"><a href="https://www.usda.gov/oig/">Office of Inspector General</a></li>
+        <li class="fs-footer-item"><a href="https://www.fs.fed.us/about-agency">About Us</a></li>
+        <li class="fs-footer-item"><a href="http://www.dm.usda.gov/foia/">FOIA</a></li>
+        <li class="fs-footer-item"><a href="https://www.eeoc.gov/eeoc/statistics/nofear/qanda.cfm">No Fear Data Act</a></li>
+        <li class="fs-footer-item"><a href="https://www.fs.fed.us/about-agency/budget-performance">Budget and Performance</a></li>
+      </ul>
+      <ul class="usa-unstyled-list">
+        <li class="fs-footer-item">(800) 832-1355</li>
+        <li class="fs-footer-item"><a href="mailto:fs-permitting-project@gsa.gov">fs-permitting-project@gsa.gov</a></li>
+      </ul>
+    </div>
+  </div>
+  <div class="usa-footer-secondary_section">
+    <div class="usa-grid">
+      <div class="usa-width-two-thirds">
+        <img src="/assets/img/usfslogo.svg" alt="US Forest Service" class="fs-footer-logo" role="img">
+        <div class="usa-footer-logo-heading">
+          <div class="fs-name-slogan">
+            <span class="fs-name">U.S. FOREST SERVICE</span>
+            <span class="fs-slogan">Caring for the land and serving people</span>
+          </div>
+        </div>
+      </div>
+      <div class="usa-footer-contact-links usa-width-one-third">
+        <a href="https://www.facebook.com/USForestService/" class="usa-link-facebook"><img src="/assets/img/social-icon-facebook.svg" alt="Facebook" target="_blank" /></a>
+        <a href="https://twitter.com/forestservice" class="usa-link-twitter"><img src="/assets/img/social-icon-twitter.svg" alt="Twitter" target="_blank" /></a>
+        <a href="https://www.youtube.com/user/usdaForestService" class="usa-link-youtube"><img src="/assets/img/social-icon-youtube.svg" alt="YouTube" target="_blank" /></a>
+      </div>
+    </div>
+    <div class="usa-grid build-version">
+      <div class="usa-width-one-whole">
+        <p>version: {{ version }}, released on: {{ buildDate |  date: 'MM/dd/yyyy'  }}</p>
```

Comment:
FYI, the date here is still saying last July - probably need to create an issue for this.